src/components/Search.js

```diff
@@ -0,0 +1,19 @@
+function Search(props) {
+  return (
+    <div>
+      <form action="/" method="get">
+        <input
+          type="text"
+          value={props.searchQuery}
+          onInput={(e) => props.setSearchQuery(e.target.value)}
+          id="search-bar"
+          placeholder="Search for food"
+          name="s"
+        />
+        <button type="submit">Search</button>
```

Comment:
We do not want to have button in this case as it refreshes the browser and your newly created itenms disappear